pages/common/nix-env.md

```diff
@@ -30,3 +30,7 @@
 - Upgrade all packages:
 
 `nix-env -u`
+
+- Install a package from a URL:
```

Comment:
Let's move this to after the "Install package" example

Reply:
Hrm, looks like Travis CI is having a hiccup moment @azuzunaga. We can force-mergethis is necessary.

Reply:
@sbrl, I rebased off the latest changes on master and force pushed to restart the travis ci build, seems to have worked 🙂